RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -147,7 +147,9 @@
 eras.trackingPhase2PU140.toReplaceWith(detachedQuadStepTrajectoryFilterBase,
     _detachedQuadStepTrajectoryFilterBase.clone(
```

Comment:
I'd suggest to remove `detachedQuadStepStandardTrajectoryFilter` and switch `detachedQuadStepTrajectoryFilter` to use `detachedQuadStepTrajectoryFilterBase` instead (as is done with `trackingPhase1PU70`; of course change the parameters of `detachedQuadStepTrajectoryFilterBase` to those of `detachedQuadStepStandardTrajectoryFilter`).